app/models/schema/common.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+##
+# Defining a common superset of attributes for Resources
+module Schema
+  module Common
+    extend ActiveSupport::Concern
+
+    ##
+    # Retrieve the attributes from Hyrax, imported from external systems, and local
+    def self.attributes
+      Plum.attributes + Schema::IIIF::Book.attributes
+    end
+
+    included do
+      Common.attributes.each do |field|
+        attribute field
```

Comment:
It's fine if it's not possible — or we could make a separate ticket for looking into that later so it doesn't block this work.